public/app/features/canvas/runtime/element.tsx

```diff
@@ -29,6 +29,11 @@ export class ElementState implements LayerElement {
   // Determine whether or not element is in motion or not (via moveable)
   isMoving = false;
 
+  // Determine whether or not to use temp stored constraint for visualization purposes
+  // (switch to top / left constraint to simplify some functionality)
+  shouldUseTempConstraint = false;
+  tempConstraint: Constraint = {};
```

Comment:
Don't love this pattern, but without it is harder to match Figma's UX (keep showing constraint visualizations on re-size, as well as keeping the layout options in sync (i.e. not flashing / changing to top / left and then back to original))